react/week5/github-search/src/routercomponent.js

```diff
@@ -0,0 +1,142 @@
+import React from "react";
+import Context from "./context";
+import "bootstrap/dist/css/bootstrap.min.css";
+import Container from "react-bootstrap/Container";
+import Row from "react-bootstrap/Row";
+import { useParams, Link } from "react-router-dom";
+import Card from "react-bootstrap/Card";
+import Column from "react-bootstrap/Col";
+
+function UserRouter() {
```

Comment:
This technically isn't a router. It's still a component, it just happens to be rendered inside of a router. It would be more appropriate to simply name this `User`